### PR TITLE
DIV-4539

### PR DESCRIPTION
**Pull request by gurpiarbassi**
-new document for co-respondent answers

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
https://tools.hmcts.net/jira/browse/DIV-4539

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by hosainnet** [comment]
Quite a few conflicts here

**Comment by gurpiarbassi** [comment]
> Quite a few conflicts here

resolved. waiting for build

File: src/integrationTest/resources/documentgenerator/documents/jsoninput/co-respondent-answers-defended-admit-costs.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            "id": "1517833758870511",
+            "case_data": {
+                "D8caseReference": "LV17D80100",
+                "D8PetitionerFirstName": "Barney",
+                "D8PetitionerLastName": "Rubble",
+                "D8RespondentFirstName": "Fred",
+                "D8RespondentLastName": "Flintstone",
+                "D8ReasonForDivorceAdultery3rdPartyFName": "Jane",
+                "D8ReasonForDivorceAdultery3rdPartyLName": "Jamed",
+                "CoRespDefendsDivorce": "YES",
```
Comment (qzhou-hmcts) [inline src/integrationTest/resources/documentgenerator/documents/jsoninput/co-respondent-answers-defended-admit-costs.json]: Capital letter

Comment (qzhou-hmcts) [reply src/integrationTest/resources/documentgenerator/documents/jsoninput/co-respondent-answers-defended-admit-costs.json]: Your are right, Uppecase!

**Comment by gurpiarbassi** [comment]
Here are a couple of templates to test it through swagger
```
{ "template" : "co-respondent-answers",
     "values": {
         "caseDetails" : {
             "id": "1517833758870511",
             "case_data": {
                 "D8caseReference": "LV17D80100",
                 "D8PetitionerFirstName": "Barney",
                 "D8PetitionerLastName": "Rubble",
                 "D8RespondentFirstName": "Fred",
                 "D8RespondentLastName": "Flintstone",
                 "D8ReasonForDivorceAdultery3rdPartyFName": "Jane",
                 "D8ReasonForDivorceAdultery3rdPartyLName": "Jamed",
                 "CoRespDefendsDivorce": "YES",
                 "CoRespAdmitAdultery": "YES",
                 "CoRespAgreeToCosts": "YES"
             }
         }
     }
 },

{ "template" : "co-respondent-answers",
     "values": {
         "caseDetails" : {
             "id": "1517833758870511",
             "case_data": {
                 "D8caseReference": "LV17D80100",
                 "D8PetitionerFirstName": "Barney",
                 "D8PetitionerLastName": "Rubble",
                 "D8RespondentFirstName": "Fred",
                 "D8RespondentLastName": "Flintstone",
                 "D8ReasonForDivorceAdultery3rdPartyFName": "Jane",
                 "D8ReasonForDivorceAdultery3rdPartyLName": "Jamed",
                 "CoRespDefendsDivorce": "NO",
                 "CoRespAdmitAdultery": "NO",
                 "CoRespAgreeToCosts": "NO",
                 "CoRespCostsReason": "some free text...."
             }
         }
     }
 }
```

File: src/test/java/uk/gov/hmcts/reform/divorce/documentgenerator/service/impl/DocumentManagementServiceImplUTest.java [hunk 2]
```diff
@@ -49,6 +49,7 @@
     private static final String AOS_INVITATION_NAME_FOR_PDF_FILE = "AOSInvitation.pdf";
     private static final String CO_RESPONDENT_INVITATION_NAME_FOR_PDF_FILE = "CoRespondentInvitation.pdf";
     private static final String RESPONDENT_ANSWERS_NAME_FOR_PDF_FILE = "RespondentAnswers.pdf";
+    private static final String CO_RESPONDENT_ANASWERS_NAME_FOR_PDF_FILE = "CoRespondentAnswers.pdf";
```
Comment (tchow8) [inline src/test/java/uk/gov/hmcts/reform/divorce/documentgenerator/service/impl/DocumentManagementServiceImplUTest.java]: CO_RESPONDENT_ANASWERS_NAME_FOR_PDF_FILE
ANASWERS typo haha :sheep:

**Comment by nsingamsetty** [comment]
This is tested and working as Expected. Refer DIV-4539 for more details.

**Comment by nsingamsetty** [comment]
Updated the correct Jira Ticket number to DIV-4539 - This is the sub task of DIV-4537

**Pull request merged**